openmdao/test/record_util.py

```diff
@@ -0,0 +1,33 @@
+import inspect
```

Comment:
I don't see a need for this but want to check with Calvin and he had to leave for the day.